spyder/plugins/ipythonconsole/widgets/client.py

```diff
@@ -785,7 +788,7 @@ def set_elapsed_time_visible(self, state):
 
     def set_info_page(self):
         """Set current info_page."""
-        if self.info_page is not None:
+        if self.infowidget is not None:

```

Comment:
```suggestion
        if self.infowidget is not None and self.info_page is not None:
```

I think the logic set here requires that.

Reply:
Makes sense.